### PR TITLE
fix: sejarah jawa barat text

### PR DESCRIPTION
**Pull request by bangunbagustapa**
### Related

- [S10A4 - Sejarah Jawa Barat](https://airtable.com/app7SdZInMN1pG6ZL/tblLy5A3qYF19fj1u/viwDUthwzD6mc5jka/recS1dCz8UjBgfXsU?blocks=hide)

### Evidence

TItle: fix: sejarah jawa barat text
Project: Portal Jabar
Participants: @bangunbagustapa @maulanayuseph @agunghide @yoslie @doohanas

### CODE REVIEW

File: pages/tentang-jawa-barat/index/sejarah-jabar.vue [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default {
           id: 5,
           year: 'Abad ke-19',
           title: 'Kekuasaan VOC Belanda Semakin Terasa di Jawa Barat',
-          desription: 'VOC mengembangkan wilayah kekuasaannya dengan mendirikan kekuasaannya di wilayah Banten. Upaya pengembangan wilayah kekuasaan tersebut yang tercatat pada tahun 1603 Masehi. Pada masa itu, Banten mengizinkan Kongsi Dangan Belanda, yakni VOC untuk menyandarkan kapal miliknya, mendirikan kantor-kantornya, hingga mengizinkan untuk mendirikan benteng pertahanannya di wilayah Banten tersebut.'
+          desription: 'VOC mengembangkan wilayah kekuasaannya dengan mendirikan kekuasaannya di wilayah Banten. Upaya pengembangan wilayah kekuasaan tersebut yang tercatat pada tahun 1603 Masehi. Pada masa itu, Banten mengizinkan Kongsi dangan Belanda, yakni VOC untuk menyandarkan kapal miliknya, mendirikan kantor-kantornya, hingga mengizinkan untuk mendirikan benteng pertahanannya di wilayah Banten tersebut.'
```
Comment (maulanayuseph) [inline pages/tentang-jawa-barat/index/sejarah-jabar.vue:68]: I think this typo should be `Kongsi dagang` 🤔 #cmiiw

**Review by maulanayuseph — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

File: pages/tentang-jawa-barat/index/sejarah-jabar.vue (hunk 1 above)
Comment (yoslie) [reply pages/tentang-jawa-barat/index/sejarah-jabar.vue:68]: I think it's `dengan` 🤣

**Review by yoslie — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

File: pages/tentang-jawa-barat/index/sejarah-jabar.vue (hunk 1 above)
Comment (bangunbagustapa) [reply pages/tentang-jawa-barat/index/sejarah-jabar.vue:68]: Wait, I have to make sure the word 😂😂 @maulanayuseph @yoslie

Comment (bangunbagustapa) [reply pages/tentang-jawa-barat/index/sejarah-jabar.vue]: Changes here https://github.com/jabardigitalservice/portal-jabar-frontend/pull/191/commits/ba2c5a864e51659a20be5d7854cef900536a779c. It should be "dengan" 😂

**Pull request merged**